<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeDeviceView</class>
 <widget class="QWidget" name="TreeDeviceView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OctoVault VPN</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="itemHolder" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-color:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/assets.qrc">:/img/images/icon_device_2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblDeviceName">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>6</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:#FFFFFF;
border-color:transparent;
}</string>
    </property>
    <property name="text">
     <string>Device Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOsName">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>22</y>
      <width>120</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:#818083;
border-color:transparent;
}</string>
    </property>
    <property name="text">
     <string>OS Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
